<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HomeScreen</class>
 <widget class="QWidget" name="HomeScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>739</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #ffffff</string>
  </property>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>579</y>
     <width>719</width>
     <height>10</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QProgressBar{
	
 	border: none;
	background-color: #ffffff
}

QProgressBar::chunk{

	background-color: #006064;
}</string>
   </property>
   <property name="value">
    <number>100</number>
   </property>
   <property name="textVisible">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="statusLabel">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>40</y>
     <width>459</width>
     <height>140</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>60</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{

	color: #000000;
}</string>
   </property>
   <property name="text">
    <string>Status</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pauseplayButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>140</width>
     <height>140</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{

	border: none;
	background-color: #006064
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../../icons.qrc">
     <normaloff>:/Icons/UI/Icons/sharp-play_arrow-24px.svg</normaloff>:/Icons/UI/Icons/sharp-play_arrow-24px.svg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>140</width>
     <height>140</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="timeLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>200</y>
     <width>659</width>
     <height>100</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>50</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{

	color: #000000;
}</string>
   </property>
   <property name="text">
    <string>19:42:42</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="eValueLabel">
   <property name="geometry">
    <rect>
     <x>596</x>
     <y>499</y>
     <width>81</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{

	border-color: #000000;
	border-radius: 0px;
	border-width: 1px;
	border-style: solid;
	color: #000000
}</string>
   </property>
   <property name="text">
    <string>488.88888</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="zValueLabel">
   <property name="geometry">
    <rect>
     <x>424</x>
     <y>499</y>
     <width>81</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{

	border-color: #000000;
	border-radius: 0px;
	border-width: 1px;
	border-style: solid;
	color: #000000
}</string>
   </property>
   <property name="text">
    <string>488.88888</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="yValueLabel">
   <property name="geometry">
    <rect>
     <x>252</x>
     <y>499</y>
     <width>81</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{

	border-color: #000000;
	border-radius: 0px;
	border-width: 1px;
	border-style: solid;
	color: #000000
}</string>
   </property>
   <property name="text">
    <string>488.88888</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="xValueLabel">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>499</y>
     <width>81</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{

	border-color: #000000;
	border-radius: 0px;
	border-width: 1px;
	border-style: solid;
	color: #000000
}</string>
   </property>
   <property name="text">
    <string>488.88888</string>
   </property>
  </widget>
  <widget class="QLabel" name="xLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>509</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{

	color: #000000;
}</string>
   </property>
   <property name="text">
    <string>X:</string>
   </property>
  </widget>
  <widget class="QLabel" name="yLabel">
   <property name="geometry">
    <rect>
     <x>222</x>
     <y>509</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{

	color: #000000;
}</string>
   </property>
   <property name="text">
    <string>Y:</string>
   </property>
  </widget>
  <widget class="QLabel" name="zLabel">
   <property name="geometry">
    <rect>
     <x>395</x>
     <y>509</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{

	color: #000000;
}</string>
   </property>
   <property name="text">
    <string>Z:</string>
   </property>
  </widget>
  <widget class="QLabel" name="eLabel">
   <property name="geometry">
    <rect>
     <x>566</x>
     <y>509</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{

	color: #000000;
}</string>
   </property>
   <property name="text">
    <string>E:</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../../icons.qrc"/>
 </resources>
 <connections/>
</ui>
